crates/schema-impl/src/lib.rs

```diff
@@ -237,9 +216,9 @@ impl Schemas {
     pub fn overwrite(
         &self,
         updates: impl IntoIterator<Item = SchemasUpdateCommand>,
-    ) -> Result<(), SchemasUpdateError> {
+    ) -> Result<(), Error> {
```

Comment:
Why is this method returning an `Error`?

Reply:
Sorry this is a leftover, i removed it.